game/src/Game/LoadingScreen.gd

```diff
@@ -0,0 +1,32 @@
+extends Control
+
+@export var progress_bar: ProgressBar
+@export var quote_label: Label
+
+var loadthread: Thread
+var quotes: PackedStringArray = []
+
+func update_loading_screen(percent_complete: int, quote_should_change = false):
+	if percent_complete > 99:
+		get_tree().change_scene_to_file("res://src/Game/GameMenu.tscn")
+		return
+	progress_bar.value = percent_complete
+	if quote_should_change:
+		quote_label.text = quotes[randi() % quotes.size()]
+
+func _on_splash_container_splash_end():
+	show()
+
+func _ready():
+	# FS-3, UI-30, UIFUN-35
+	loadthread = Thread.new()
+	loadthread.start(Events.load_events)
```

Comment:
```suggestion
	loadthread.start(Events.load_events.bind(self))
```
Given the thread the `LoadingScreen` that queued it, instead of using the hardcoded nodepath in `load_events`.